hvplot/interactive.py

```diff
@@ -365,6 +365,22 @@ def get_ax():
         return get_ax
 
     def __call__(self, *args, **kwargs):
+        """
```

Comment:
See comment in post about whether the docstring should be on the Class, `__init__` or `__call__` level.